### DISCUSSION TITLE
I can't get MotionEye to run when RASPAP is installed

### DISCUSSION THREAD

**Discussion by evandene** [RaspAP/raspap-webgui#1000] [Q&A]
My setup:
Raspberry pi4 4Gb, RASPAP headless installed according instructions via WiFi and HSS
For a moment I had a setup RASPAP running as AP-STA mode but after a reboot I couldn't get it to run in AP-STA mode.  I tried many things but nothing did work.
Anyway, I moved on and installed MotionEye but can't get is to run as I had planned to do.

TODAY:
MotionEye is used by me in rural areas without phone or internet connection. Till now I have with me a router fired up by a powerbank and in that way I can connect my RPI's to the router and STREAM my video's to tablets or smartphones. (No internet needed)

I'm in the process now, skipping the router and using the RPI4 as an AP-STA station that runs MotionEye what than streams video to my tablets, also without internet connection.
It looks that I can use some help here because I'm out of idea's myself

**Comment by billz** [comment]
For AP-STA to startup after a reboot the `raspapd.service` must be installed and active. However, external WiFi adapters will almost always give you better performance and stability than AP-STA. They are also cheap and ubiquitous. See [this FAQ](https://docs.raspap.com/faq/#where-can-i-find-a-list-of-usb-wifi-adapters-that-use-in-kernel-drivers) for a list of recommended USB WiFi adapters that use in-kernel drivers.

Re: MotionEye, never used it so cannot advise you. Suggest you start a discussion or raise an issue on their repo for support.
